Testing `test/e2e/alcotest-lwt/failing/async_failure.ml'.
This run has ID `<uuid>'.

> [FAIL]    all › one
  [PASS]    all › two

┌──────────────────────────────────────────────────────────────────────────────┐
│ [FAIL]    all › one                                                          │
└──────────────────────────────────────────────────────────────────────────────┘
freeing all resources
[failure] All is broken
          <stacktrace>
          
Logs saved to `<build-context>/_build/_tests/<test-dir>/all/one.output'.
 ──────────────────────────────────────────────────────────────────────────────

Full test results in `<build-context>/_build/_tests/<test-dir>'.
1 failure! in <test-duration>s. 2 tests run.
